test/Sema/object_literals_osx.swift

```diff
@@ -22,3 +22,5 @@ struct Path: _ExpressibleByFileReferenceLiteral {
 
 let p1: Path = #fileLiteral(resourceName: "what.txt")
 let p2 = #fileLiteral(resourceName: "what.txt") // expected-error{{could not infer type of file reference literal}} expected-note{{import Foundation to use 'URL' as the default file reference literal type}}
+
+let text = #fileLiteral(resourceName: "TextFile.txt").relativeString! // expected-error{{type of expression is ambiguous without more context}}
```

Comment:
Is this actually ambiguous, or is this because there is no relativeString member in any of the things that #fileLiteral could be and we're emitting a misleading diagnostic?